tests/test_suite.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python2
+from __future__ import print_function
+
+import argparse
+import os
+import os.path
+import sys
+import time
+
+from lib.log import log
```

Comment:
changed it to `ssg_test_suite`. I have also tried abbreviation `ssgts` or something like that, to make it shorter, but it was not easy to type :)